content/departments/engineering/dev/process/principles-and-practices.md

```diff
@@ -29,3 +29,46 @@ If you have an idea for something you want to work on, then you have a few optio
 
 1. **Just do it** if you can timebox or spread out the effort such that it won't impact your ability to deliver on existing plans.
 2. If working on your idea would require a non-trivial amount of time or would impact your ability to deliver on existing plans, then have a discussion with your manager to come up with a feasible plan (for example: explicitly schedule time to work on your idea during the next iteration).
+
+## Ownership
+
+Ownership at Sourcegraph, as exercised by the most successful engineering teams and individuals here, is built on _high agency_ and _customer focus_.
+
+High agency because ownership at Sourcegraph doesn't come with a lot of "how"s attached. Instead you get a lot of freedom. Freedom for you to decide how exactly you want to reach your goals.
+
+Customer focus because ownership here (again: as exercised by the most successful teams and individuals) is not solely about code ownership. It's about owning the **responsibility of providing value to customers in the area you own.** It's about outcomes, not output.
+
+It's not this:
+
+> I own X because I work on tickets that are labeled with `team-X`
+
+No. It's this:
+
+> I own X because my goal is to solve problem X for customers and I'll do everything I can as an engineer to make that happen.
+
+And yes, that comes with _responsibility_ -- with high agency and a customer-first attitude comes high responsibility. Everything else flows from that:
+
+- It's up to _you_ to make sure your work meets engineering standards.
+- It's up to _you_ to make sure it doesn't blow up in production.
+- It's up to _you_ to sometimes push back against feature requests or a roadmap, to make sure there's enough time to take care of engineering concerns, like refactoring and testing.
+- It's up to _you_ to come up with ideas on how to improve what you're building.
+- It's up to _you_ to think about whether what you build actually solves the customer's problem and provides value. You, as an engineer, can see possibilities and options that others can't – it's up to you to make use of that knowledge.
+- It's up to _you_ to collaborate with product, design, and analytics to ensure there is a business case for your work, it meets our user experience standards, and that analytics exist to help gauge its value to the customer.
+- It's up to _you_ to define a roadmap if one is missing and to then execute on it, to update it, and to share what you learned with others, so they can understand what you're working on and why and how it relates to their work.
+- It's up to _you_ to propose decisions that you think should be made. Instead of waiting and asking for others to make decisions for you and being blocked. And if you don't get the feedback on your proposed decisions that you need, it's up to you to let others know that you're going to execute on what you decided and then execute.
```

Comment:
```suggestion
- It's up to _you_ to propose decisions that you think should be made, instead of waiting and asking for others to make decisions for you while you are blocked. And if you don't get the feedback on your proposed decisions that you need, it's up to you to let others know that you're going to execute on what you decided and then execute.
```
nit: minor fix, but up to you

Reply:
+1 on this suggestion (my english-parser went off and I had to backtrack)